README.md

```diff
@@ -85,23 +85,21 @@
 
 - Absatz 3. Ränge wie "Premium" sind nur bei der Administratoren, den Inhabern oder im offiziellen Shop erhältlich. Bitte fallt nicht auf andere Shops, Verkäufer Via TeamSpeak, Skype und Co. rein! Das Vortäuschen eines Rangkaufs ist eine Straftat welche nach § 263 StGB unter Betrug fällt und schwer bestraft wird und zur Strafanzeige gebracht werden kann.
 
-# § 3 Besondere Regelungen für TeamSpeak
+# § 3 Besondere Regelungen für TeamSpeak & Discord
 
 - Absatz 1. Avatare & Namen
 
-  - Absatz 1.1. Das Nutzen von Klammern wie () [] {} ist nicht erlaubt. Genauso ist das Nutzen von jeglichen Sonderzeichen mit der Ausnahme von | und @ nicht erlaubt.
```

Comment:
Wieso sollten wir diese Regel streichen? Damit wäre es erlaubt sich `[Admin]` vor den Namen zu schreiben. Zumindest sollte dieser Teil angepasst werden, dass das "faken" von Rängen nicht erlaubt ist.

Reply:
Auf unserem TeamSpeak ist dies sowieso schon gesperrt und auf Discord kann nicht umbenannt werden. Des Weiteren werden hier jegliche Sonderzeichen verboten, Spieler verwenden aber welche und dies auch mit Bedacht auf unserem TeamSpeak. Daher sah ich diese Regel für unnötig

Reply:
Bots können ausfallen. Um dafür bannen zu können muss es in den Regeln manifestiert sein

Reply:
Du hast Recht, dann benötigt diese Regel aber eine andere Formulierung und anderen Inhalt. Denn ich denke nicht, dass das Verbieten von jeglichen Sonderzeichen nötig ist. Ich mache mich an die Umformulierung sobald ich am PC bin.